mc-conflicts-ui/src/conflicts/models/conflict-search-params.spec.ts

```diff
@@ -0,0 +1,89 @@
+import { ConflictSearchParams } from './conflict-search-params';
+import { Geometry } from 'geojson';
+
+const geom: Geometry = {
+  type: 'Polygon',
+  coordinates: [
+    [
+      [0, 0],
+      [0, 1],
+      [1, 1],
+      [1, 0],
+      [0, 0],
+    ],
+  ],
+};
+
+it('isDateRangeValid returns true when both dates are undefined', () => {
+  const store = ConflictSearchParams.create({});
+
+  expect(store.isDateRangeValid).toEqual(true);
+});
+
+it('isDateRangeValid returns true if only one date is defined', () => {
+  const store = ConflictSearchParams.create({ from: new Date() });
+
+  expect(store.isDateRangeValid).toEqual(true);
+});
+
+it('isDateRangeValid returns false if both dates are defined, but from > to', () => {
+  const store = ConflictSearchParams.create({
+    from: new Date(1595222000),
+    to: new Date(1595221000),
```

Comment:
it is unclear from first sight what make the isDateRangeValid, 
I would write something like:
```
const epochTimestamp = 1595222000;
const fromDate = new Date(epochTimestamp);
const toDate = new Date(epochTimestamp);
toDate.setDate(toDate.getDate() - 5);
const store = ConflictSearchParams.create({
    from: new Date(1595222000),
    to: new Date(1595221000),
});
```
now it would be much clearer that `to` is set five days before `from`

Reply:
fixed in b2e85dc